source/ci_source/_tests/_travis.test.js

```diff
@@ -39,7 +39,7 @@ describe(".isPR", () => {
     env[key] = null
 
     test(`does not validate when ${key} is missing`, () => {
-      const travis = new Travis({})
+      const travis = new Travis(env)
```

Comment:
The Travis CI source test wasn't validating the `env` where `env[key]` was null. Updated this test to do so (which I think it intends to do).